scheduler/src/cook/config.clj

```diff
@@ -408,17 +408,18 @@
                                             :launch-wait-seconds (get fitness-calculator :launch-wait-seconds 60)
                                             :update-interval-ms (get fitness-calculator :update-interval-ms nil)}))
      :plugins (fnk [[:config {plugins {}}]]
-                (let [{:keys [job-launch-filter job-submission-validator]} plugins]
-                  (merge plugins
-                         {:job-launch-filter
-                          (merge
-                            {:age-out-last-seen-deadline-minutes 10
-                             :age-out-first-seen-deadline-minutes 600
-                             :age-out-seen-count 10}
-                            job-launch-filter)
-                          :job-submission-validator
-                          (merge {:batch-timeout-seconds 40}
-                                 job-submission-validator)})))}))
+                   (-> plugins
+                       (update :job-launch-filter
+                                  (fn [job-launch-filter]
+                                    (merge
+                                     {:age-out-last-seen-deadline-minutes 10
+                                      :age-out-first-seen-deadline-minutes 600
+                                      :age-out-seen-count 10}
+                                     job-launch-filter)))
+                       (update :job-submission-validator
+                               (fn [job-submission-validator]
+                                 (merge {:batch-timeout-seconds 40}
+                                        job-submission-validator)))))}))
```

Comment:
Is this change needed?

Reply:
No, I think this was changed while I was working on this PR. Reverted.